CONTRIBUTING.md

```diff
@@ -95,35 +78,35 @@ $ git add my/changed/files
 $ git commit
 ```
 
-### Commit guidelines
-
-Writing good commit logs is important. A commit log should describe what
-changed and why. Follow these guidelines when writing one:
```

Comment:
“A commit log should describe what changed and why” is still good advice, at least in theory…